### ISSUE TITLE
[QUESTION] Updating sapling strategy

### ISSUE THREAD

**Issue by fgsch** [facebook/sapling#272]
Hi. The installation instructions at https://sapling-scm.com/docs/introduction/installation don't talk about updates, presumably having to download new binaries every time there is a release.

Are there any plans to address this? For example, for Homebrew users you could create a tap.

**Comment by sggutier** [comment]
We are currently trying to get merged into Homebrew-core directly

**Comment by nemith** [comment]
Even a tap would be better right now, right?  https://github.com/facebook/homebrew-fb already exists?

**Comment by bolinfest** [comment]
@nemith It does, but we would rather reduce the number of times we change the installation instructions.

**Comment by bolinfest** [comment]
We're still waiting on a response on our homebrew-core PR if you know anyone: https://github.com/Homebrew/homebrew-core/pull/116334

**Comment by mhlakhani** [comment]
@bolinfest : it looks like that PR has landed - are there any updates here? :)

(hoping to have to update less frequently, I really love using Sapling so far)

**Comment by bolinfest** [comment]
@mhlakhani oops, I guess we forgot to close this! Yes, we are in Homebrew core, so `brew install sapling` is all you need on macOS now. I just pushed a new release yesterday!

The installation docs have already been updated to reflect this.

**Comment by bolinfest** [comment]
I guess this is closed for macOS via Homebrew, but not Linux or Windows. Those probably merit their own GitHub issues?

**Comment by mhlakhani** [comment]
@bolinfest thanks! do I need to do anything to remove my pre-existing version (installed via Homebrew) before I run that command, or will it just work?
